flake.nix

```diff
@@ -0,0 +1,57 @@
+# This flake is supposed to enable a convenient development environment.
+# It is independent of any packaging in nixpkgs.
+#
+# See https://github.com/limine-bootloader/limine/issues/330 for more details
+# regarding the packaging in nixpkgs.
+
+{
+  description = "limeboot";
```

Comment:
Limeboot?
```suggestion
  description = "limine";
```

Reply:
Yikes, thanks. This is the part that I didn't check a second or a third time. 😂